kong/plugins/aws-lambda/api-gateway/aws/AWSBasicCredentials.lua

```diff
@@ -0,0 +1,42 @@
+--[[
+  Copyright 2016 Adobe Systems Incorporated. All rights reserved.
```

Comment:
@sinzone See new comments on #1005 for a new PR imminent which switches to a different non-adobe MIT-licensed lib.